### DISCUSSION TITLE
Disable `delete after pair feature`

### DISCUSSION THREAD

**Discussion by hungpham3112** [LunarWatcher/auto-pairs#62] [Q&A] [answered]
As doc:

```
2. Features                                                 *autopairs-features*

Insert in pair: >

        input: [
        output: [|]

Delete in pair: >

        input: foo[<BS>]
        output: foo

Delete after pair: >
        input: foo[]<BS>
        output: foo
<
Note: This feature is disabled by default, and must be explicitly enabled.
See |g:AutoPairsMapBS|
```
I just want to use the first and second feature, but `let g:AutoPairsMapBS=1` also enables `delete after pairs` feature. How can I turn off it. Thanks.

**Comment by LunarWatcher** [comment, marked as answer]
I assume you mean `let g:AutoPairsMapBS=1`?

 that aside, there's no way to disable that specific type of deletion at the moment. It's not hard to add it as an option, though, but that's a feature request 

---

This has now been implemented; use `let g:AutoPairsBSAfter = 0` to disable deletion after pair. Still requires `let g:AutoPairsMapBS = 1`. See `:h g:AutoPairsBSAfter`and `:h g:AutoPairsBSIn`

**Reply by hungpham3112** [reply]
> I assume you mean `let g:AutoPairsMapBS=1`?
> 
> that aside, there's no way to disable that specific type of deletion at the moment. It's not hard to add it as an option, though, but that's a feature request

I fixed a typo.

**Comment by eirnym** [comment]
I vote for this feature request as well, to be able to enable or disable each option separately

**Reply by LunarWatcher** [reply]
Finally got my device up and running; I'm planning to set this up within the next few days. No guarantees on exactly when, because life. When it does roll out, it'll be enabled by default (though only work if MapBS = 1, for obvious reasons)

**Reply by hungpham3112** [reply]
Any update for this feature?

**Reply by LunarWatcher** [reply]
See the second half of my initial post

**Reply by hungpham3112** [reply]
> See the second half of my initial post

Thanks a lot
